cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -288,7 +288,7 @@ The following commands are only executable against the `system` database:
 |===
 |Command                                                                                                                                                                            | Admin category | Description
 | <<administration-security-users-alter-password, ALTER CURRENT USER SET PASSWORD FROM ... TO>>                                                                                     | User and role  | Change the password of the user that is currently logged in.
-| <<administration-security-users-alter, ALTER USER ... [SET [PLAINTEXT \| ENCRYPTED] PASSWORD {password [CHANGE [NOT] REQUIRED] \| CHANGE [NOT] REQUIRED}] [SET STATUS {ACTIVE \| SUSPENDED}]>>             | User and role  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
+| <<administration-security-users-alter, ALTER USER ... [IF EXISTS] [SET [PLAINTEXT \| ENCRYPTED] PASSWORD {password [CHANGE [NOT] REQUIRED] \| CHANGE [NOT] REQUIRED}] [SET STATUS {ACTIVE \| SUSPENDED}]>>             | User and role  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
```

Comment:
Still think this looks horrible and should be fixed in general, but that's a different discussion...

Reply:
I agree on that